tests/unmount.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { defineComponent } from 'vue'
+
+import { mount } from '../src'
+
+const AXIOM = 'Rem is the best girl'
+
+describe('Unmount', () => {
+  it('works on single root component', () => {
+    const errorHandler = jest.fn()
+    const Component = {
+      template: `
+        <div>${AXIOM}</div>
+      `
+    }
+    const wrapper = mount(Component, {
+      props: {},
+      global: {
+        config: {
+          errorHandler
+        }
+      }
+    } as any) // The type checking keeps complaning about unmatched type which might be a bug
+    wrapper.unmount()
+    expect(errorHandler).not.toHaveBeenCalled()
+  })
+
+  it('works on multi-root component', () => {
+    const errorHandler = jest.fn()
+    const Component = defineComponent({
+      template: `
+        <div>${AXIOM}</div>
```

Comment:
I don't think we need `AXIOM`, it doesn't really add much to the test. I am not sure who Rem is.

Can we just have `<div /><div />` or something?

Reply:
Yep, sure